cmd/ndm_daemonset/probe/probe.go

```diff
@@ -32,7 +32,9 @@ var RegisteredProbes = []func(){
 	smartProbeRegister,
 	mountProbeRegister,
 	udevProbeRegister,
-	sysfsProbeRegister}
+	sysfsProbeRegister,
+	usedbyProbeRegister,
```

Comment:
maybe rename it to consumerProbe if it makes sense.

Reply:
Discussed with @pawanpraka1 and @akhilerm over slack. Keeping this as used-by probe for now.